### ISSUE TITLE
Thanks

### ISSUE THREAD

**Issue by riahc3** [JustOscarJ1/valorant_lineup_calculator#7]
In part of your program, Valorant has changed the ping system.

So

This program does not work anymore...but at least, cheaters cant cheat anymore.

Well done

**Comment by riahc3** [comment]
> Ain't no way this guy was mad for a whole year and came back to post this. In your head rent free man... it's not that deep.

😂 Little bro doesn't know that activity on GitHub can be seen on the upper right corner and casually I saw this.

What's worse is you replying to me, thinking people care. Make a alt and boost your own ego 

Dude so mad Riot removed his cheating tool that he thinks he is special 😂😂😂😂😂😂 

Calm down chosen one 😂

**Comment by JustOscarJ1** [comment]
I genuinely do not understand why you think I'm mad. All I'm doing is pointing out your obsessiveness and grudge from when I made this, which caused you to make some whiney issue with fake communications from Riot, then stop responding when I called you out, and finally come back to post this.

I made this for fun, and I haven't played Val for like 6 months, if others got use from this I'm happy and don't care it got patched.

I really find it unlikely you happened to stumble upon some nonexistent notification the day the patch came, but sure man, you're so nonchalant and cool stumbling upon it, really got me! also, "What's worse is you replying to me, thinking people care. Make a alt and boost your own ego." Really sorry to tell you man, but you're the one who made this issue with no motivation. I was just responding to your random attack.

move on with your life
